### ISSUE TITLE
Release soon?

### ISSUE THREAD

**Issue by mvdan** [mastodon/mastodon-android#360]
Hello, and thanks for your work!

I notice that a lot of fixes and some new features have happened since the last release in July. I also notice that my Mastodon experience is a bit buggier on mobile than on web, probably due to this.

Is there a chance that you could do a release soon? :)

**Comment by grishka** [comment]
Issues are not for questions like this. The next release is whenever @Gargron says.

**Comment by mvdan** [comment]
Thanks. To give a bit of perspective, there is literally no other way for me to contact the project if I navigate to your github repo. No discussions, no forum, no messaging platform. Opening an issue was really the only reasonable option I had.

**Comment by thekalinga** [comment]
(not sure if anyone else has already asked this qn before, if yes, pls excuse my ignorance & point me a link where I can learn more about. I went over main mastodon website & I couldnt find much on the governance part of mastodon)

Other than @Gargron deciding the future, is there any actual community participation in the major decisions being made to this project?

I know its an opensource project (I've contributed to other projects & built my own reasonably complex opensource code in the past) & he is a major contributor to project (appreciate for he has done for project), but given the importance Mastodon have for many in society & community, isnt it important to ensure relevant people (founders, contributors, funders & people who use it — at times some wear more than one hat) also can participate in decision making process in somewhat transparent manner?

If more people feel part of the process, they will invest time and energy. Whats important is not just development effort (tho its an important leg just like other important leg — being community participation), but also those who have no clue about the project (as they are new to mastodon — people like me) can get their qns answered by volunteers who become part of the ecosystem.
